solution-jivanov/.gitignore

```diff
@@ -0,0 +1,12 @@
+# package directories
+node_modules
+jspm_packages
+
+# Serverless directories
+.serverless
+
+# IDE folders
+.idea
+
+# Other folders
+coverage/
```

Comment:
Do you tend to use linting/formatting tools standardizing line-endings?

Reply:
Oooh, yes, I've seen "wars" between senior devs on whether to use single or double quotes :) Certainly, lining is very useful and can catch a lot of issues. My IDE of choice (WebStorm/PhpStorm) has built-in linting and I rely on it every day. If you guys have any particular rules on endings, etc. please let me know, will be happy to adjust my IDE.